vault/resource_ldap_secret_backend_test.go

```diff
@@ -61,6 +61,13 @@ func TestLDAPSecretBackend(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, consts.FieldConnectionTimeout, "30"),
 				),
 			},
+			{
+				SkipFunc: func() (bool, error) {
+					return !testProvider.Meta().(*provider.ProviderMeta).IsAPISupported(provider.VaultVersion116), nil
+				},
+				Config: testLDAPSecretBackendConfig_withSkip(path, bindDN, bindPass),
+				Check:  resource.TestCheckResourceAttrSet(resourceName, consts.FieldSkipStaticRoleImportRotation),
```

Comment:
Can we check that this is the value we expect?
```suggestion
				Check:  resource.TestCheckResourceAttr(resourceName, consts.FieldSkipStaticRoleImportRotation, "true"),
```